orchagent/pfcactionhandler.cpp

```diff
@@ -404,7 +404,7 @@ bool PfcWdLossyHandler::getHwCounters(PfcWdHwStats& counters)
         return false;
     }
 
-    sai_object_id_t pg = portInstance.m_priority_group_ids[getQueueId()];
+    sai_object_id_t pg = portInstance.m_priority_group_ids[static_cast <size_t> (getQueueId())];
```

Comment:
>static_cast <size_t> (getQueueId())]; [](start = 59, length = 37)

Could you provide the compilation message on original code?

Reply:
Error log:
```
pfcactionhandler.cpp:408:70: error: conversion to 'std::vector<long long unsigned int>::size_type {aka unsigned int}' from 'sai_object_id_t {aka long long unsigned int}' may alter its value [-Werror=conversion]
     sai_object_id_t pg = portInstance.m_priority_group_ids[getQueueId()];
                                                            ~~~~~~~~~~^~
```